pkg/yang/entry.go

```diff
@@ -647,7 +647,9 @@ func ToEntry(n Node) (e *Entry) {
 				if e.RPC == nil {
 					e.RPC = &RPCEntry{}
 				}
-				e.RPC.Input = ToEntry(i)
+				in := ToEntry(i)
+				in.Parent = e
+				e.RPC.Input = in
```

Comment:
Alternative: I wasn't as sure about directly assigning 
```
  e.Dir["input"] = in
```
at this point; that would avoid the additional `e.RPC != nil` case during `Find`, below by just making `e.Dir` track the schema node identifiers in this case; `input` and `output` are valid `schema-node-identifier` elements.

Thoughts?  If we're meant to see child identifiers as keys in `Dir`, it seems correct to add the `'input` or `'output` key to `e.Dir` in the code block here.

Reply:
I think this should work.  It is a breaking change since the RPC field of Entry would go away.  It is not clear if we would need to add an RPCEntry kind to distinguish RPC entries from other containers.  I will approve this change as it is a non-breaking change.  Using Dir instead of RPC can be decided upon after this goes in.

Reply:
Sounds good, thanks for the comments.